backend/scripts/nlp/lambda_function.py

```diff
@@ -0,0 +1,56 @@
+""" This is a copy of the Lambda function that was uploaded to AWS, it is not intended to run locally
+and it will not work without modification"""
+
+import json
+import nltk
+import requests
+nltk.download('punkt')
+
+
+
+
+
+def lambda_handler(event=None, context=None):
+    sentence = "Who am I"
+    # sentence = event["sentence"]
+    
+    print(sentence)
+    tokens = nltk.word_tokenize(sentence.lower())
+    tokenlist = ''
+    firstword = tokens[0]
+    print(firstword)
+    
+    isQuestion = False
+
+    
+    
+    for s in tokens:
+        tokenlist += s
+
+    print(tokenlist)
+        
+    if '?' in tokenlist:
+        isQuestion = True
+    
+    wh_question = ['what', 'when', 'where', 'who',
+                    'whom', 'which', 'whose', 'why', 'how']
+    yN_question = ["am", "is", "are", "do", "does", "did", "have", "has", "was", "were", "can", "cannot", "could",
+                    "couldn't", "dare", "may", "might", "must", "need", "ought", "shall", "should", "shouldn't", "will", "would"]
+    
+    if firstword in wh_question or firstword in yN_question:
+        isQuestion = True
+    print(isQuestion)
+    
+    answer = ''
```

Comment:
It looks like there's a difference between the lambda code and what's shown here. If this is the lambda code clean u lines 45-55

Reply:
Yes, there was a difference between the code here and in AWS, I used the local one for testing. Anyway, I updated the code.